### PR TITLE
Remove redundant dockerfile entrypoint

### PR DESCRIPTION
**Pull request by quoctruong**
Remove the debug entrypoint as we are now using skaffold debug.

### CODE REVIEW

File: python/python-hello-world/src/Dockerfile [hunk 1]
```diff
@@ -14,4 +14,4 @@ RUN pip install --trusted-host pypi.python.org -r requirements.txt
 COPY . .
 
 # Run app.py when the container launches
-ENTRYPOINT ["python", "-m", "ptvsd", "--port", "3000", "--host", "0.0.0.0", "app.py"]
```
Comment (matthewmichihara) [inline python/python-hello-world/src/Dockerfile:17]: Oh nice these aren't required anymore? I think we can also clean up the `Procfile`s as well

**Comment by matthewmichihara** [comment]
@quoctruong can you update the `Procfile`s as well?

**Comment by quoctruong** [comment]
@matthewmichihara Sure. I'll do that in another PR

**Pull request merged**